### ISSUE TITLE
move back from sveltekit to svelte

### ISSUE THREAD

**Issue by aryadaroui** [aryadaroui/Favy#15]

**Comment by 0djek** [comment]
If there is a plan to add more pages, I think keeping `SvelteKit` would be more beneficial for the ease of creating new pages. If there only be one/two pages, going to `Svelte` would be better. What do you think?

**Comment by aryadaroui** [comment]
> If there is a plan to add more pages, I think keeping `SvelteKit` would be more beneficial for the ease of creating new pages. If there only be one/two pages, going to `Svelte` would be better. What do you think?

Totally. As the app currently, stands, it doesnt make sense to be using SvelteKit because it's just a single view. But I've been considering setting a foundation for 3rd party plugins in case  people want to process their photos on export. I could be thinking too far ahead though. idk

**Comment by 0djek** [comment]
I also saw some starting points of the settings page, so maybe it's better to stay on SvelteKit and see how it will go? If it looks like it's really not needed, SvelteKit could be discarded and Svelte implemented

**Comment by aryadaroui** [comment]
Yeah the settings page is going to become a modal like the export button is. I should that to the kanban 😬
